movie/form.js

```diff
@@ -19,7 +20,7 @@ export function render(movie) {
         <input type="text" id="year" name="year" value="${movie.year}" />
       </div>
         <label for="public">Public:</label>
-        <input type="checkbox" id ="public" name = "public" value="1" checked />
+        <input type="checkbox" id ="public" name = "public" value="public" ${checked}/>
```

Comment:
What does is return if the the box is not checked?
I assume it's null, for this reason I would also save it here as an boolean.

Reply:
do you mean the `value="public"` part or the `${checked}` part ?
The first one is because "public" is a way better value then "1", its best if you try it out and check out the Web Request data sent from the Browser. 

The Second thing is for editing, if a movie was public before, this checkbox will be checked, If it wasnt, that checkbox will not be checked.